code/game/atoms.dm

```diff
@@ -36,7 +36,7 @@ var/global/list/ghdel_profiling = list()
 	//Detective Work, used for the duplicate data points kept in the scanners
 	var/list/original_atom
 
-	var/list/beams
+	var/list/beams = list()

```

Comment:
This is horrible you just added a list to every single atom in the game without mentioning it, and say even for a moment you did NEED this (which I doubt) you would need to modify the atom level resetvariables for it